aea/aea.py

```diff
@@ -82,9 +82,7 @@ def __init__(
 
         self.max_reactions = max_reactions
         self._task_manager = TaskManager()
-        self._decision_maker = DecisionMaker(
-            self.name, self.max_reactions, self.outbox, wallet, ledger_apis
-        )
+        self._decision_maker = DecisionMaker(identity, wallet, ledger_apis)
```

Comment:
The DM should not directly send envelopes to the outbox, and hasn't used it anyway, so it is now removed.